### PR TITLE
Add CONTRIBUTING.md and DEVELOPING.md files

### PR DESCRIPTION
**Pull request by sbrunner**
This is based on the Openlayers one, what do we do with the 'Asking Questions'?

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+# Contributing to c2cgeoportal
+
+Thanks for your interest in contributing to a GeoMapFish project.
+
+## Asking Questions
+
+Please ask questions about using the library on [Stack Overflow using the tag 'openlayers']
```
Comment (gberaudo) [inline CONTRIBUTING.md]: Maybe not "openlayers" ;)
Is someone of the team monitoring SW?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+(http://stackoverflow.com/questions/tagged/openlayers).
+
+When you want to get involved and discuss new features or changes, please use [the mailing list]
+(https://groups.google.com/forum/#!forum/openlayers-dev).
```
Comment (gberaudo) [inline CONTRIBUTING.md]: Idem

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+
+## Submitting Bug Reports
+
+Please use the [GitHub issue tracker](https://github.com/camptocamp/c2cgeoportal/issues). Before creating a
+new issue, do a quick search to see if the problem has been reported already.
```
Comment (gberaudo) [inline CONTRIBUTING.md]: Greate we are using github issues and not JIRA.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+## Contributing Code
+
+See [`DEVELOPING.md`](https://github.com/camptocamp/c2cgeoportal/blob/master/DEVELOPING.md) to learn how to
+get started developing.
+
+Our preferred means of receiving contributions is through [pull requests]
+(https://help.github.com/articles/using-pull-requests). Make sure that your pull request follows our pull
+request guidelines below before submitting it.
+
+This page describes what you need to know to contribute code to OpenLayers as a developer.
```
Comment (gberaudo) [inline CONTRIBUTING.md]: idem

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
+# Contributing to c2cgeoportal
+
+Thanks for your interest in contributing to a GeoMapFish project.
```
Comment (jwkaltz) [inline CONTRIBUTING.md]: to a -> to the

File: CONTRIBUTING.md (hunk 1 above)
Comment (jwkaltz) [inline CONTRIBUTING.md]: the library -> the OpenLayers library

File: DEVELOPING.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
+# Developing
+
+To develop in the admin interface read the relater [README]
```
Comment (gberaudo) [inline DEVELOPING.md]:
```suggestion
To develop in the admin interface read the [README]
```

File: DEVELOPING.md [hunk 7]
```diff
@@ -0,0 +1,10 @@
...
+(https://github.com/camptocamp/c2cgeoportal/blob/master/admin/README.md).
+
+To develop in the geoportal part you should [create a new application]
+(https://camptocamp.github.io/c2cgeoportal/master/integrator/create_application.html),
+Then have a loog on the [application debuging]
```
Comment (gberaudo) [inline DEVELOPING.md]:
```suggestion
Then have a look on the [application debugging]
```

File: CONTRIBUTING.md (hunk 2 above)
Comment (jwkaltz) [inline CONTRIBUTING.md]: Why do we mention the openlayers-dev forum here? Copy-paste mistake?

File: CONTRIBUTING.md (hunk 4 above)
Comment (jwkaltz) [inline CONTRIBUTING.md]: GeoMapFish?

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+(https://help.github.com/articles/using-pull-requests). Make sure that your pull request follows our pull
+request guidelines below before submitting it.
+
+This page describes what you need to know to contribute code to OpenLayers as a developer.
+
+
+## Contributor License Agreement
+
+Your contribution will be under our [license]
+(https://raw.githubusercontent.com/camptocamp/c2cgeoportap/master/LICENSE.md) as per
```
Comment (jwkaltz) [inline CONTRIBUTING.md]: c2cgeoportap -> c2cgeoportal

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+The commit history explains to the reviewer the series of modifications to the code that you have made and
+breaks the overall contribution into a series of easily-understandable chunks.  Any individual commit should
+not add more than one new class or one new function.  Do not submit commits that change thousands of lines
+or that contain more than one distinct logical change.  Trivial commits, e.g. to fix lint errors, should be
+merged into the commit that introduced the error.  See the [Atomic Commit Convention on Wikipedia]
+(http://en.wikipedia.org/wiki/Atomic_commit#Atomic_Commit_Convention) for more detail.
+
+`git apply --patch` and `git rebase` can help you create a clean commit history.
+[Reviewboard.org](http://www.reviewboard.org/docs/codebase/dev/git/clean-commits/)
+and [Pro GIT](http://git-scm.com/book/en/Git-Tools-Rewriting-History) have explain how to use them.
```
Comment (jwkaltz) [inline CONTRIBUTING.md]: have explain -> explain

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -0,0 +1,101 @@
...
+
+
+### Use clear commit messages
+
+Commit messages should be short, begin with a verb in the imperative, and
+contain no trailing punctuation. We follow
+http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html
+for the formatting of commit messages.
+
+Git commit message should look like:
```
Comment (jwkaltz) [inline CONTRIBUTING.md]: message -> messages

File: DEVELOPING.md (hunk 6 above)
Comment (jwkaltz) [inline DEVELOPING.md]: relater -> related

File: DEVELOPING.md (hunk 7 above)
Comment (jwkaltz) [inline DEVELOPING.md]: loog on the -> look at the

debuging -> debugging

File: CONTRIBUTING.md (hunk 1 above)
Comment (sbrunner) [reply CONTRIBUTING.md]: To be discussed, @ybolognini

File: CONTRIBUTING.md (hunk 2 above)
Comment (sbrunner) [reply CONTRIBUTING.md]: To be discussed, @ybolognini

File: CONTRIBUTING.md (hunk 3 above)
Comment (sbrunner) [reply CONTRIBUTING.md]: To be discussed, @ybolognini

File: CONTRIBUTING.md (hunk 4 above)
Comment (sbrunner) [reply CONTRIBUTING.md]: Thanks, fixed :-)

File: CONTRIBUTING.md (hunk 1 above)
Comment (sbrunner) [reply CONTRIBUTING.md]: To be discussed, @ybolognini

File: CONTRIBUTING.md (hunk 2 above)
Comment (sbrunner) [reply CONTRIBUTING.md]: To be discussed, @ybolognini

**Comment by sbrunner** [comment]
@gberaudo Yes we need this CONTRIBUTIN.md, it's a requirement of OSGeo :-)

File: DEVELOPING.md [hunk 11]
```diff
@@ -0,0 +1,10 @@
+# Developing
+
+To develop in the admin interface read the related [README]
+(https://github.com/camptocamp/c2cgeoportal/blob/master/admin/README.md).
+
+To develop in the geoportal part you should [create a new application]
+(https://camptocamp.github.io/c2cgeoportal/master/integrator/create_application.html),
+Then have a loog on the [application debugging]
```
Comment (jwkaltz) [inline DEVELOPING.md]: loog -> look

**Pull request merged**